### PR TITLE
[CI] Update to XCode 11.4 and tools to 5.2

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 5.0.
- [ ] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -12,11 +12,9 @@ jobs:
   Darwin:
     name: Darwin
     runs-on: macos-latest
-    strategy:
-      matrix:
-        xcode: ['/Applications/Xcode_11.3.app/Contents/Developer']
```
Comment (guykogus) [inline .github/workflows/CI.yml:17]: You can't reuse this?

Comment (LucianoPAlmeida) [reply .github/workflows/CI.yml:17]: I think we don't really need a matrix here ... we just need to specify environment

Comment (guykogus) [reply .github/workflows/CI.yml:17]: I meant more as a reusable element. Not sure what a strategy matrix is, I haven't looked into GitHub workflows yet.

Comment (LucianoPAlmeida) [reply .github/workflows/CI.yml:17]: Yeah, I just change this

**Pull request merged**